### PR TITLE
KO: translation for site/ko/guide/keras/functional.ipynb

### PR DESCRIPTION
**Pull request by krorina1013**
translated functional.ipynb
EN -> KO

EN: https://github.com/tensorflow/docs/blob/master/site/en/guide/keras/functional.ipynb

### CODE REVIEW

**Comment by krorina1013** [comment]
> xed any issues), please reply here with
@googlebot I signed it!

File: site/ko/guide/keras/functional.ipynb [hunk 1]
```diff
@@ -0,0 +1,1493 @@
...
+    "```\n",
+    "\n",
+    "#### It validates your model while you're defining it.\n",
+    "\n",
+    "함수형 API에서는 입력 사양(shape 및 dtype)이 미리 입력되어 있으며 (`Input`을 통해), 층를 호출 할 때마다 해당 층은 층을 통과하는 사양이 해당 가정과 일치하는지 확인합니다. 그렇지 않으면 유용한 오류 메세지를 제공합니다.\n",
+    "\n",
+    "이를 통해 Functional API로 빌드 할 수있는 모든 모델이 실행됩니다. 컨버젼스 관련 디버깅 이외의 모든 디버깅은 실행 시간이 아니라 모델 생성 중에 정적으로 발생합니다. 이것은 컴파일러의 유형 검사와 유사합니다.\n",
+    "\n",
+    "\n",
+    "#### 함수형 모델은 구성하거나 검사할 수 있습니다.\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: 구성하거나 --> 그래프로 출력하거나

File: site/ko/guide/keras/functional.ipynb [hunk 2]
```diff
@@ -0,0 +1,1493 @@
...
+    "\n",
+    "모델은 그래프로 구성할 수 있으며, 예를 들어, 이전 예시에서 보았듯이, 중간 층의 활성화를 출추하고 재사용하기위해 쉽게 그래프의 중간 노드에 접근할 수 있습니다:\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: 모델은 그래프로 구성할 수 있으며, 예를 들어, 이전 예시에서 보았듯이, 중간 층의 활성화를 출추하고 재사용하기위해
-->
모델은 그래프로 그릴 수 있습니다. 그리고 예를 들어, 이전 예시에서 보았듯이 중간 층의 활성화를 추출하고 재사용하기 위해

File: site/ko/guide/keras/functional.ipynb [hunk 3]
```diff
@@ -0,0 +1,1493 @@
...
+    "\n",
+    "```python\n",
+    "features_list = [layer.output for layer in vgg19.layers]\n",
+    "feat_extraction_model = keras.Model(inputs=vgg19.input, outputs=features_list)\n",
+    "```\n",
+    "\n",
+    "\n",
+    "#### Your Functional model can be serialized or cloned.\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: Your Functional model can be serialized or cloned
-->
함수형 모델을 직렬화하거나 복사할 수 있습니다

File: site/ko/guide/keras/functional.ipynb [hunk 4]
```diff
@@ -0,0 +1,1493 @@
...
+    "\n",
+    "함수형 모델은 코드가 아닌 데이터 구조이기 때문에 안전하게 직렬화 할 수 있으며 단일 코드로 저장하여 원본 코드에 액세스하지 않고도 정확히 동일한 모델을 다시 만들 수 있습니다. 자세한 내용은 [saving and serialization guide](./save_and_serialize.ipynb)를 참조하십시오.\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: 단일 코드로 --> 하나의 파일로

File: site/ko/guide/keras/functional.ipynb [hunk 5]
```diff
@@ -0,0 +1,1493 @@
...
+    "\n",
+    "\n",
+    "### 함수형 API의 약점 :\n",
+    "\n",
+    "#### 동적 아키텍처를 지원하지 않습니다.\n",
+    "\n",
+    "\n",
+    "함수형 API는 모델을 층의 DAG로 처리합니다. 이는 대부분의 딥 러닝 아키텍처에 해당하지만 전부는 아닙니다. 예를 들어 재귀 네트워크 또는 RNN 트리에서는 이 가정을 따르지 않으며 함수형 API에서 구현할 수 없습니다.\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: 딥 러닝 --> 딥러닝

File: site/ko/guide/keras/functional.ipynb [hunk 6]
```diff
@@ -0,0 +1,1493 @@
...
+    "\n",
+    "\n",
+    "#### 때때로, 처음부터 모든 것을 작성해야할 필요가 있습니다.\n",
+    "\n",
+    "고급 아키텍처를 작성할 때 \"층의 DAG 정의\"범위를 벗어난 작업을 수행 할 수 있습니다. 예를 들어 모델 인스턴스에서 다중 사용자 정의 학습 및 인터페이스 방법을 노출 할 수 있습니다. 이 경우에는 하위 클래스 작업이 필요합니다\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: 예를 들어 모델 인스턴스에서 다중 사용자 정의 학습 및 인터페이스 방법을 노출 할 수 있습니다.
-->
예를 들어 모델 인스턴스에 여러 개의 사용자 정의 훈련 및 추론 메서드를 제공해야 할 수 있습니다.

File: site/ko/guide/keras/functional.ipynb [hunk 7]
```diff
@@ -0,0 +1,1493 @@
...
+    "  아키텍처를 작성시에, 층 DAG 정의 영역 밖의 것을 하고 싶어할 지도 모른다. 예를 들어, 다중 사용자 정의 훈련과 인터페이스 방법\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: Please remove this line.

File: site/ko/guide/keras/functional.ipynb [hunk 8]
```diff
@@ -0,0 +1,1493 @@
...
+    "\n",
+    "\n",
+    "---\n",
+    "\n",
+    " 함수형 API와 모델 하위 클래스 사이의 차이점에 대해 더 자세히 알고싶다면 다음을 읽어보시기 바랍니다.\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: 읽어보시기 바랍니다. --> 읽어 보기 바랍니다.

File: site/ko/guide/keras/functional.ipynb [hunk 9]
```diff
@@ -0,0 +1,1493 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Ym1jrCqusGvj"
+   },
+   "source": [
+    "## 서로 다른 API 스타일 혼합 및 매칭\n",
+    "\n",
+    " 중요한 것은, 기능 API 또는 모델 하위 분류 중 하나를 선택하는 것은 한 범주의 모델로 제한하는 이항 결정이 아니라는 점입니다. tf.keras API의 모든 모델은 sequentail 모델, 함수형 모델 또는 처음부터 작성된 하위 분류 모델/층 등 각 모델과 상호 작용할 수 있습니다.\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: 'binary decision' is hard to translate.
I suggest to skip 'binary' like 이항 결정이 --> 결정이

sequentail  --> Sequential

File: site/ko/guide/keras/functional.ipynb [hunk 10]
```diff
@@ -0,0 +1,1493 @@
...
+    "\n",
+    " 또한 함수형 모델 또는 시퀀셜 모델을 하위 분류 모델/층의 한 부분으로 사용할 수 있습니다:"
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: 시퀀셜 --> Sequential

File: site/ko/guide/keras/functional.ipynb [hunk 11]
```diff
@@ -0,0 +1,1493 @@
...
+    "\n",
+    "\n",
+    "class CustomRNN(layers.Layer):\n",
+    "\n",
+    "  def __init__(self):\n",
+    "    super(CustomRNN, self).__init__()\n",
+    "    self.units = units\n",
+    "    self.projection_1 = layers.Dense(units=units, activation='tanh')\n",
+    "    self.projection_2 = layers.Dense(units=units, activation='tanh')\n",
+    "    # 이전 정의 함수형 모델\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: 이전 정의 --> 이전에 정의된

File: site/ko/guide/keras/functional.ipynb [hunk 12]
```diff
@@ -0,0 +1,1493 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "oxW1d0a8_ufg"
+   },
+   "source": [
+    " 반대로, 다음과 같이 `call` mothod를 구현하면, subclassed Layer 또는 함수형 API의 모델을 이용할 수 있습니다:\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: 다음과 같이 `call` mothod를 구현하면, subclassed Layer 또는 함수형 API의 모델을 이용할 수 있습니다
-->
다음과 같은 패턴 중 하나를 따르는 `call` 메서드를 구현하면 함수형 API에서 하위 클래스로 만든 어떤 층이나 모델도 사용할 수 있습니다.

File: site/ko/guide/keras/functional.ipynb [hunk 13]
```diff
@@ -0,0 +1,1493 @@
...
+    "\n",
+    "\n",
+    "- `call(self, inputs, **kwargs)`  는  `inputs` 이 텐서이거나 텐서가 중첩된 구조이며,(e.g. 텐서 목록), `**kwargs`이 텐서가 아닌 인수(비입력)입니다.\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: `call(self, inputs, **kwargs)`  는 `inputs`이 --> `call(self, inputs, **kwargs)`에서 `inputs`는

e.g. --> 예를 들어

`**kwargs`이 텐서가 아닌 인수(비입력)입니다. --> `**kwargs`는 (`inputs`을 제외한) 텐서가 아닌 매개변수입니다.

File: site/ko/guide/keras/functional.ipynb [hunk 14]
```diff
@@ -0,0 +1,1493 @@
...
+    "- `call(self, inputs, training=None, **kwargs)` 는 `training`이 층이 훈련 모드 및 추론 모드에서 작동해야하는지 여부를 나타내는 부울입니다.\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: `call(self, inputs, training=None, **kwargs)` 는 `training`이 --> `call(self, inputs, training=None, **kwargs)`에서 `training`은
부울입니다 --> 불리언(boolean) 매개변수입니다.

File: site/ko/guide/keras/functional.ipynb [hunk 15]
```diff
@@ -0,0 +1,1493 @@
...
+    "- `call(self, inputs, mask=None, **kwargs)` 는 `mask`가 부울 마스크 텐서입니다 (예를 들어, RNN에 유용합니다).\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: `call(self, inputs, mask=None, **kwargs)` 는 `mask`가 부울
-->
`call(self, inputs, mask=None, **kwargs)`에서 `mask`는 불리언

File: site/ko/guide/keras/functional.ipynb [hunk 16]
```diff
@@ -0,0 +1,1493 @@
...
+    "- `call(self, inputs, training=None, mask=None, **kwargs)` -- 물론 마스킹과 특정 행동을 훈련하는 것을 동시에 가질 수 있습니다.\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: 특정 행동을 훈련하는 것을 동시에 가질 --> 특정 훈련 행동을 동시에 지정할

File: site/ko/guide/keras/functional.ipynb [hunk 17]
```diff
@@ -0,0 +1,1493 @@
...
+    "\n",
+    " 또한, `get_config`방법을 사용자 지정 층 또는 모델에서 구현 할 경우, 이를 사용하여 생성한 함수형 모델은 직렬화하거나 복사할 수 있습니다. \n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: `get_config`방법을 사용자 지정 층 --> `get_config` 메소드를 사용자 정의 층

File: site/ko/guide/keras/functional.ipynb [hunk 18]
```diff
@@ -0,0 +1,1493 @@
...
+    "\n",
+    " 처음부터 함수형 모델에서 작성된 사용자 정의 RNN을 사용한 예시입니다:"
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: 처음부터 함수형 모델에서 작성된 사용자 정의 RNN을 
-->
함수형 모델에서 처음부터 작성된 사용자 정의 RNN을

File: site/ko/guide/keras/functional.ipynb [hunk 19]
```diff
@@ -0,0 +1,1493 @@
...
+    "      x = inputs[:, t, :]\n",
+    "      h = self.projection_1(x)\n",
+    "      y = h + self.projection_2(state)\n",
+    "      state = y\n",
+    "      outputs.append(y)\n",
+    "    features = tf.stack(outputs, axis=1)\n",
+    "    return self.classifier(features)\n",
+    "\n",
+    "# `CustomRNN`의 내부 연산은 정적 배치 크기를 요구하기 때문에 `batch_shape`인수\n",
+    "# 에 대한 정적 배치 크기를 지정하다는 것을 유의하십시오.\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: `batch_shape`인수 --> `batch_shape` 매개변수
지정하다는 것을 유의하십시오 --> 지정한다는 것을 유의하세요

File: site/ko/guide/keras/functional.ipynb [hunk 20]
```diff
@@ -0,0 +1,1493 @@
...
+    "# (`state`가 zeros tensor일 때).\n",
```
Comment (rickiepark) [inline site/ko/guide/keras/functional.ipynb]: zeros tensor --> 0 텐서

**Comment by lamberta** [comment]
Thank you.

Community translations are now located in the [tensorflow/docs-l10n](https://github.com/tensorflow/docs-l10n) repo (see [this message](https://groups.google.com/a/tensorflow.org/g/docs/c/GJPq7S_fw4c/m/lA1qfBX2EQAJ) for context).

I'll move over/duplicate existing pull requests as I find time. In the meantime, please feel free to create a new pull request in that repo 😀 Then you can close this PR and add a link to the new one.
